rflx/identifier.py

```diff
@@ -25,7 +25,7 @@ def __init__(
 
         error = RecordFluxError()
         if not self._parts:
-            error.extend([("empty identifier", Subsystem.ID, Severity.ERROR, location)])
+            fatal_fail("empty identifier", Subsystem.ID, Severity.ERROR, location)
         elif "" in self._parts:
             error.extend(
                 [(f'empty part in identifier "{self}"', Subsystem.ID, Severity.ERROR, location)]
```

Comment:
Using `FatalError` only for this one error case seems inconsistent to me. If we consider an empty identifier as a fatal error, all other forms of invalid identifiers should be also considered fatal.

Reply:
I agree. Changed in b972d7aa.